src/nanoemoji/nanoemoji.py

```diff
@@ -270,15 +295,25 @@ def write_variable_font_build(nw: ninja_syntax.Writer, font_config: FontConfig):
     nw.newline()
 
 
-def _run(argv):
-    os.makedirs(build_dir(), exist_ok=True)
-
+def _font_config_for_build(cli_srcs):
     # Dump config with defaults, CLI args, etc resolved to build
-    cli_srcs = tuple(Path(f) for f in argv if f.endswith(".svg"))
-    font_config = config.load(additional_srcs=cli_srcs)
+    # and sources updated to point to build picosvgs
+    original_config = config.load(additional_srcs=cli_srcs)
+    font_config = _update_sources(original_config)
+
+    # Save it so we can point write_font at it later
     config_file = build_dir() / "config.toml"
     config.write(config_file, font_config)
-    print("Wrote ", rel_build(config_file))
+    print("Wrote ", config_file)
+    return original_config
```

Comment:
wait, so this function has a side effect of saving an updated config file in the build_dir (pointing to picosvg sources), but then returns the original config? I find that a bit confusing. Maybe split the loading of the original from the updating/writing of the modified one?

Reply:
Ugh ty; that function got revised into horror